wnghub/controller/view.py

```diff
@@ -14,8 +15,24 @@ class BaseNotificationViewController(BaseController, ABC):
     stdout nor formatting the table.
     """
 
+    _exclude_url_size = 90
+
+    _exclude_repo_type_size = 97
+
+    _exclude_repo_size = 114
+
+    _exclude_type_size = 121
+
+    _excluded_for_terminal = False
```

Comment:
State variables are risky here, probably only in theory but still. This object might get shared btwn future threads, etc.

Reply:
I left the excluded for terminal state variable here now. We can change that in the future/rework the expand terminal message in the future if needed.